Searching_Sorting/ShellSort.py

```diff
@@ -0,0 +1,26 @@
+def shellSort(input_list):
+    gap = len(input_list) // 2

```

Comment:
you can save len(input_list) here outside and reuse it as len(input_list)) is the same across the function right?